aws/cloudformation-templates/deployment-support.yaml

```diff
@@ -503,3 +503,95 @@ Resources:
         Fn::GetAtt: 
           - "PinpointPreCreateRule"
           - "Arn"
+
+  ####################### Pre-Create Pinpoint SMS Alerts Lambda #######################
+  PinpointSMSAlertsLambda:
+    Condition: DeployPreCreatePinpointWorkshop
```

Comment:
Good point! I got it confused with being similar to PinpointAutoWorkshop lambda. I will remove the condition, since the lambda needs to be present for the customer to run it using the workshop.